src/routes/claims/+page.server.ts

```
@@ -0,0 +1,91 @@
+import { redirect, type Actions } from '@sveltejs/kit';
+import type { PageServerLoad } from './$types';
+import { getPendingClaims, getMyClaims, approveClaim, rejectClaim, getObjetosByIds } from '$lib/api';
+
+async function enrichClaimsWithObjetos(claims: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)